src/OrchardCore.Themes/TheTheme/Views/Layout.cshtml

```diff
@@ -36,6 +37,7 @@
             @await RenderBodyAsync()
         </div>
     </main>
+    @await RenderSectionAsync("Footer", required: false)
```

Comment:
What do we have in the footer? Or is it just for consistency with the other themes?

Reply:
Yes, I'd like all the themes have the same zones.